featuretools/computational_backends/feature_set.py

```diff
@@ -7,6 +7,7 @@
 from featuretools.feature_base import (
     AggregationFeature,
     GroupByTransformFeature,
+    MultiOutputFeature,
```

Comment:
maybe we can name this better.  Something that indicates it just one of the outputs of a multi output feature

what about `FeatureOutputDimension`?